pkg/cmd/util.go

```diff
@@ -702,3 +703,20 @@ func updateFlagsToOptions(interactive, skipPreview, yes bool) (backend.UpdateOpt
 		SkipPreview: skipPreview,
 	}, nil
 }
+
+// loadJSONConfigFile loads file with json configuration data and unmarshals into a map[string]*json.RawMessage object.
```

Comment:
Since this is in a generic util file, maybe be more specific about its use in the comment? Or just move it to the pkg/cmd/policy_enable.go file. I dont think its used anywhere else?

Reply:
Ok, I went ahead and relocated to policy_enable.co and then leveraged the function justin is suggesting since it does some validations.